### PR TITLE
feat(ui): revamp user profile page

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- I worked on revamping user profile page
- added unit test for user profile components
- fix the profile icon and name if display name not provided.

image
image
image
image

image

Issue : 
image

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Comment by Ashish8689** [comment]
code smell is showing in the permission check so we can avoid this.

File: openmetadata-ui/src/main/resources/ui/src/components/Users/Users.component.tsx [hunk 1]
```diff
@@ -224,97 +278,72 @@ const Users = ({
...
+          bordered={false}
+          className="header-collapse-custom-collapse user-profile-container"
+          expandIconPosition="end">
+          <Collapse.Panel
+            className="header-collapse-custom-panel"
+            collapsible="icon"
+            header={userProfileCollapseHeader}
+            key="1">
+            <Row>
+              <Col className="border-top p-y-lg" span={24}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Users/Users.component.tsx:293]: can we use gutter prop of Row instead adding padding?

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/Users/Users.component.tsx]: @chirag-madlani  changes made as per comments, we still have added some padding around the cell as required as per the designs!

**Review by chirag-madlani — APPROVED** [review]

**Comment by chirag-madlani** [comment]
@Ashish8689 Can you check 2 code smells?

**Comment by Ashish8689** [comment]
> @Ashish8689 Can you check 2 code smells?

here it is https://github.com/open-metadata/OpenMetadata/pull/13744#issuecomment-1786658579

**Pull request merged**